test/parallel/test-os-userinfo-handles-getter-errors.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
```

Comment:
This appears (in GitHub) to be new rather than renamed?

Reply:
@richardlau Yes, I had to split `test-regress-GH-12371` into two tests to properly rename it. Thinking about it, I should probably have done that when I originally wrote the regression test for #12371.